src/js/api/auth/logout.test.js

```diff
@@ -0,0 +1,10 @@
+import "jest-localstorage-mock";
+import { logout } from "./logout";
+
+describe("logout", () => {
+  it("it checks if it clears browser storage on logout", () => {
+    logout();
```

Comment:
If no token was set before, this will always pass - set the token manually first!

Reply:
I have updated it to check that storage.remove is called after calling logout